vespalib/src/vespa/vespalib/test/nexus.h

```diff
@@ -23,27 +23,34 @@ concept nexus_thread_entry = requires(Nexus &ctx, T &&entry) {
 class Nexus
 {
 private:
-    vespalib::test::ThreadMeets::Vote _vote;
-    static size_t &my_thread_id();
+    using vote_t = vespalib::test::ThreadMeets::Vote;
+    vote_t &_vote;
+    size_t _thread_id;
+    Nexus(vote_t &vote, size_t thread_id) noexcept
+      : _vote(vote), _thread_id(thread_id) {}
+    Nexus(Nexus &&) = delete;
+    Nexus(const Nexus &) = delete;
+    Nexus &operator=(Nexus &&) = delete;
+    Nexus &operator=(const Nexus &) = delete;
```

Comment:
Deleted functions could be `public` to just emit compiler errors that these are explicitly deleted instead of giving a more generic private access error on top.